roles/gerrit/templates/gerrit.config.j2

```diff
@@ -0,0 +1,60 @@
+## {{ ansible_managed }}
+[gerrit]
+        basePath = git
+        canonicalWebUrl = https://{{ gerrit_url }}/
+        canonicalGitUrl = git://{{ gerrit_url }}/
```

Comment:
That's the current config, but I am wondering on the security of offering a git transport, since that's in cleartext, someone doing MITM could inject code (and then, that's a rather straightforward code execution if someone compile it).